internal/proxy/proxy.go

```diff
@@ -43,9 +43,13 @@ func New(opts *Options) (*SSOProxy, error) {
 			return nil, err
 		}
 
+		setSessionStore := SetCookieStore
+		if opts.RedisConnectionURL != "" || opts.RedisUseSentinel {
+			setSessionStore = SetRedisStore
+		}
```

Comment:
This is nice and concise, but I wonder if it's worth standardising to match the other similar use cases above, e.g:
```
if opts.RedisConnectionURL != "" || opts.RedisUseSentinel {
    optFuncs = append(optFuncs, SetRedisStore(opts)
} else {
    optFuncs = append(optFuncs, SetCookieStore(opts)
}
```
Or, at least closer to whatever method we end up using on the authenticator side to set the session store.